src/app/generic-listing-page/generic-listing-page.component.html

```diff
@@ -1,10 +1,27 @@
-<ion-content>
+<app-application-header [config]="headerConfig" (actionClick)="handleActionClick($event)">
+<ng-template #search>
   @if(pageConfig.enableSearch){
-    <!-- To be replaced with search component -->
-    <div>SEARCH BAR</div>
+    <ion-row class="srch">
+      <ion-col size="12">
+        <ion-searchbar placeholder="Hinted search text" [debounce]="1000"
```

Comment:
placeholder text also should come from en.json

Reply:
done